### ISSUE TITLE
Automated release notifications and tweets

### ISSUE THREAD

**Issue by bdougie** [netlify/netlify-cms#1036]
**- Do you want to request a *feature* or report a *bug*?**
Feature

**- What is the current behavior?**
When release gets cut, they have the possibility to be unnoticed or not tweeted. 

**- What is the expected behavior?**
I recently started working with Netlify Functions and wrote one that auto-tweets and blogs when a release is created. I am curious if the CMS team is interested in this. I realize there is no blog on the netlifycms.org, but this would be a great addition to the site somewhere.

I am working on a tutorial/blog post soon, but here is the function.

https://github.com/bdougie/create-react-app-lambda/blob/master/src/lambda/gitHubEventListener.js 

What are your thoughts on this?

**Comment by erquhart** [comment]
This is an awesome proposal, definitely on board.

**Comment by tech4him1** [comment]
Awesome, I think it's a great idea to use the platforms that we already have in place (website/Gitter). On Gitter, we can keep doing `@/all` announcements for the community meetings and new releases, I've even been missing those :).

**Comment by bdougie** [comment]
@tech4him1 true that, definitely makes sense to share that info there in an automated way. I have not used the Gitter API, but can definitely look into that. I have some time this Friday blocked out to do some R&D, I can open a PR to get eyes on it then. 

https://developer.gitter.im/docs/welcome

**Comment by bdougie** [comment]
Looks like there is a bit of crossover with #1045

**Comment by tech4him1** [comment]
@bdougie Agreed, I intended that issue to be for notifying users from inside the CMS, whereas this issue is for external notifications (tweets, etc). Feel free to combine them (or we can edit #1045 to clarify).

**Comment by martinjagodic** [comment]
Closing in favor of #1036
